skimage/segmentation/_clear_border.py

```diff
@@ -21,7 +19,7 @@ def clear_border(labels, buffer_size=0, bgval=0, in_place=False):
 
     Returns
     -------
-    labels : (N, M) array
+    labels : (M[, N[, ..., P]]) array
         Cleared binary image.
```

Comment:
I'll be very grateful if you could change this. In the current redaction the descriptions are too confusing.
If not, I'll file another issue for this.